shift_detector/checks/simple_check.py

```diff
@@ -22,13 +22,13 @@ def __init__(self, categorical_threshold=0.05, mean_threshold=0.15, median_thres
                                   'num_distinct': num_distinct_threshold, 'std': std_threshold, 'completeness':
                                   completeness_threshold}
 
-        if categorical_threshold < 0 or categorical_threshold > 1:
-            raise ValueError('The categorical threshold of {} is not correct. It has be between the values of '
+        if categorical_threshold < 0:
+            raise ValueError('The categorical threshold of {} is not correct. It has to be between the values of '
```

Comment:
You are just testing for one direction now. This is not consistent with the error message...

Reply:
weird, on my machine this is already corrected. could you re-check?
<img width="820" alt="Screen Shot 2019-06-21 at 15 15 33" src="https://user-images.githubusercontent.com/24995121/59925060-852de380-9437-11e9-80bc-0637ae264626.png">

Reply:
Thats not what I mean. When `categorical_threshold` is set to 1.5, no error is raised, but the ErrorMessage suggests this

Reply:
Ah, got you